dnf5/main.cpp

```diff
@@ -933,6 +933,14 @@ int main(int argc, char * argv[]) try {
     loggers.emplace_back(std::make_unique<libdnf5::MemoryBufferLogger>(max_log_items_to_keep, prealloc_log_items));
 
     loggers.front()->info("DNF5 start");
+    std::string cmdline;
+    for (int i = 0; i < argc; ++i) {
+        if (i > 0) {
+            cmdline += " ";
+        }
+        cmdline += argv[i];
+    }
+    loggers.front()->info("DNF5 command line: {}", cmdline);
```

Comment:
What about to merge both logging events? "DNF5 start" plus "DNF5 command line: {}".
What about new message: `DNF5 performing command: {}`. Or it would be possible to remove logging "DNF5 start" and keep the new message `DNF5 command line: {}`. I see only one reason to keep both - when formatting of  message `"DNF5 command line: {}"` fails. What do you think?

Reply:
Makes sense. I'll refactor the code to write only one message. I will also remove the code duplicity with command line arguments formatting.

Reply:
Done.